autoload/wintabs.vim

```diff
@@ -558,15 +557,7 @@ endfunction
 
 " delete buffer from buflist if it isn't attached to any wintab
 function! s:purge(buffer)
```

Comment:
let's remove the changes on this file

Reply:
Whoops, didn't mean to include these. Thanks.